tests/Controllers/AccountControllerTest.php

```diff
@@ -64,6 +67,26 @@ public function testInvalidIdsInProfileOrder()
             ->assertJsonFragment(['profile_order' => $newOrder]);
     }
 
+    /**
+     * @dataProvider dataProviderForUpdateCountry
+     *
+     * More complete tests are done through CountryChange and CountryChangeTarget.
+     */
+    public function testUpdateCountry(int $months, bool $success): void
```

Comment:
I think this test case should explicitly be testing against the countries matching (instead of assigning number of months) and leave the number of months testing to the other tests